### PR TITLE
Updates to MOSS sub-site

### PR DESCRIPTION
**Pull request by mehan**
## Description

Updates to text and links on MOSS pages to reflect changes in the program.

### CODE REVIEW

File: bedrock/mozorg/templates/mozorg/moss/index-092018.html [hunk 1]
```diff
@@ -229,7 +218,7 @@ <h2 class="moss-section-heading">{{ _('Frequently Asked Questions') }}</h2>
     <dd>
       <p>
       {% trans %}
-        Historically, the MOSS Committee has mostly chosen to fund projects which can demonstrate some success and momentum with regard to acquiring users and building an active community of contributors. If you are a new project that is just starting out or if you don’t yet have reliable information on how many users/contributors you have, we encourage you to clearly articulate your plans for growing your user base and contributor community.
+        Historically, the MOSS Committee has chosen to fund projects which are important in their ecosystems and can demonstrate some success and momentum with regard to acquiring users and building an active community of contributors. Unfortunately, the MOSS Committee does not consider applications from brand new projects, projects which are not open source, projects which have yet to be built or projects which have yet to build a community of users and contributors. If you are unsure if your project qualifies for MOSS funding, please <a href="{{ moss_email }}">email the MOSS team</a>.
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/moss/index-092018.html]: Missing the email address here

File: bedrock/mozorg/templates/mozorg/moss/mission-partners-india.html [hunk 2]
```diff
@@ -21,18 +21,15 @@
   <h2 class="moss-section-heading">{{ _('About') }}</h2>
 
   <p class="moss-copy">
-  {% trans url1=url('mozorg.about.manifesto') %}
-    The Global Mission Partners: India track is open to any open source or
-    free software project that is undertaking an activity which significantly
-    furthers Mozilla’s <a href="{{ url1 }}">mission</a>, and for which the
-    applicant(s) are based in India.
+  {% trans url1=url('mission-partners.html') %}
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/moss/mission-partners-india.html]: This is causing the page to throw an error:

```
Reverse for 'mission-partners.html' not found. 'mission-partners.html' is not a valid view function or pattern name.
```

```suggestion
  {% trans url1=url('mozorg.moss.mission-partners') %}
```

File: bedrock/mozorg/templates/mozorg/moss/mission-partners-india.html [hunk 3]
```diff
@@ -21,18 +21,15 @@
...
+    <strong>The Global Mission Partners: India track is now closed.</strong> All applications which would previously have been submitted to the Global Mission Partners: India track will now be considered under the Mission Partners track. Please see the <a href="{{ url1 }}">Mission Partners page</a> for more information.
   {% endtrans %}
   </p>
 
   <div class="moss-ctas">
     <p class="moss-copy">{{ _('Ready to go?') }}</p>
 
-    <a href="https://docs.google.com/forms/d/e/1FAIpQLSdHq-tzjUjtx-jqHd1muy1PGX9m5CO7kepl5n6_-ID2mOKi2A/viewform" rel="external" class="button">{{ _('Apply Now') }}</a>
+    <a href="mission-partners.html" rel="external" class="button">{{ _('Apply Now') }}</a>
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/moss/mission-partners-india.html]:
```suggestion
    <a href="{{ url('mozorg.moss.mission-partners') }}" rel="external" class="button">{{ _('Apply Now') }}</a>
```

File: bedrock/mozorg/templates/mozorg/moss/mission-partners-india.html [hunk 4]
```diff
@@ -204,17 +201,7 @@ <h2 class="moss-section-heading">{{ _('How to apply') }}</h2>
...
-
-  {% trans url1='https://wiki.mozilla.org/MOSS#Application_Deadlines' %}
-    For deadline information, please check
-    <a href="{{ url1 }}">the MOSS wiki</a>.
-  {% endtrans %}
-
-  {% trans url1='https://wiki.mozilla.org/MOSS/Mentors' %}
-    If you need help completing your application, see the
-    <a href="{{ url1 }}">list of mentors</a>.
+    Please see the <a href="mission-partners.html">Mission Partners page</a> for more information on how to apply.
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/moss/mission-partners-india.html]:
```suggestion
    Please see the <a href="{{ partners }}">Mission Partners page</a> for more information on how to apply.
```

File: bedrock/mozorg/templates/mozorg/moss/mission-partners-india.html [hunk 5]
```diff
@@ -204,17 +201,7 @@ <h2 class="moss-section-heading">{{ _('How to apply') }}</h2>
 
   <p class="moss-copy">
   {% trans %}
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/moss/mission-partners-india.html]:
```suggestion
  {% trans partners=url('mozorg.moss.mission-partners') %}
```

File: bedrock/mozorg/templates/mozorg/moss/mission-partners-india.html [hunk 6]
```diff
@@ -223,7 +210,7 @@ <h2 class="moss-section-heading">{{ _('How to apply') }}</h2>
       {{ _('I’m ready!') }}
     </p>
 
-    <a href="https://docs.google.com/forms/d/e/1FAIpQLSdHq-tzjUjtx-jqHd1muy1PGX9m5CO7kepl5n6_-ID2mOKi2A/viewform" rel="external" class="button">{{ _('Apply Now') }}</a>
+    <a href="mission-partners.html" rel="external" class="button">{{ _('Apply Now') }}</a>
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/moss/mission-partners-india.html]:
```suggestion
    <a href="{{ url('mozorg.moss.mission-partners') }}" rel="external" class="button">{{ _('Apply Now') }}</a>
```

**Comment by mehan** [comment]
Thanks for the help with these template errors, @alexgibson! This should now be ready for review again.

File: bedrock/mozorg/templates/mozorg/moss/index-092018.html [hunk 7]
```diff
@@ -77,24 +76,14 @@ <h2 class="moss-section-heading">{{ _('Award Selection') }}</h2>
...
-  </p>
 </section>
 
 <section class="moss-section">
   <h2 class="moss-section-heading">{{ _('Past Recipients') }}</h2>
 
   <p class="moss-copy">
   {% trans %}
-    Past MOSS awardees include:
+    Past MOSS awardees include the projects listed below. For more information on past MOSS awards, please see the <a href="https://github.com/mozilla/MOSS-Awards-Directory">MOSS awards directory</a> on GitHub.
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/moss/index-092018.html]: Please pass `href` values as variables via the `{% trans %}` block, like we do for other copy blocks. Link destinations should not be part of the translatable strings.

File: bedrock/mozorg/templates/mozorg/moss/index-092018.html [hunk 8]
```diff
@@ -229,7 +218,7 @@ <h2 class="moss-section-heading">{{ _('Frequently Asked Questions') }}</h2>
     <dd>
       <p>
       {% trans %}
-        Historically, the MOSS Committee has mostly chosen to fund projects which can demonstrate some success and momentum with regard to acquiring users and building an active community of contributors. If you are a new project that is just starting out or if you don’t yet have reliable information on how many users/contributors you have, we encourage you to clearly articulate your plans for growing your user base and contributor community.
+        Historically, the MOSS Committee has chosen to fund projects which are important in their ecosystems and can demonstrate some success and momentum with regard to acquiring users and building an active community of contributors. Unfortunately, the MOSS Committee does not consider applications from brand new projects, projects which are not open source, projects which have yet to be built or projects which have yet to build a community of users and contributors. If you are unsure if your project qualifies for MOSS funding, please <a href="mailto:moss@mozilla.com">email the MOSS team</a>.
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/moss/index-092018.html]: Same as comment above.

File: bedrock/mozorg/templates/mozorg/moss/foundational-technology.html [hunk 9]
```diff
@@ -295,14 +295,14 @@ <h2 class="moss-section-heading">{{ _('How to apply') }}</h2>
     To get started with your application, simply follow the link below.
   {% endtrans %}
 
-  {% trans url1='https://wiki.mozilla.org/MOSS#Application_Deadlines' %}
-    For deadline information, please check
-    <a href="{{ url1 }}">the MOSS wiki</a>.
+  {% trans %}
+    MOSS applications are considered by the committee on a rolling basis each month; typically, you can expect to hear back from us within two months of submitting your application.
+
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/moss/foundational-technology.html]: Nit: remove blank line

File: bedrock/mozorg/templates/mozorg/moss/mission-partners.html [hunk 10]
```diff
@@ -206,14 +206,14 @@ <h2 class="moss-section-heading">{{ _('How to apply') }}</h2>
     To get started with your application, simply follow the link below.
   {% endtrans %}
 
-  {% trans url1='https://wiki.mozilla.org/MOSS#Application_Deadlines' %}
-    For deadline information, please check
-    <a href="{{ url1 }}">the MOSS wiki</a>.
+  {% trans %}
+    MOSS applications are considered by the committee on a rolling basis each month; typically, you can expect to hear back from us within two months of submitting your application.
+
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/moss/mission-partners.html]: Nit: remove blank line

File: bedrock/mozorg/templates/mozorg/moss/mission-partners.html [hunk 11]
```diff
@@ -206,14 +206,14 @@ <h2 class="moss-section-heading">{{ _('How to apply') }}</h2>
...
+    MOSS applications are considered by the committee on a rolling basis each month; typically, you can expect to hear back from us within two months of submitting your application.
+    
   {% endtrans %}
 
-  {% trans url1='https://wiki.mozilla.org/MOSS/Mentors' %}
-    If you need help completing your application, see the
-    <a href="{{ url1 }}">list of mentors</a>.
+  {% trans %}
+    If you have any questions regarding your application, please 
+    <a href="mailto:moss@mozilla.com">email the MOSS team</a>.
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/moss/mission-partners.html]: Again, use a variable please.

**Comment by alexgibson** [comment]
@mehan friendly ping, any updates here?

**Comment by mehan** [comment]
Hi @alexgibson--apologies, have been busy with prep for MozFest--will plan to get to this this week. Thanks for the reminder!

**Comment by mehan** [comment]
@alexgibson Apologies again for the delay on this--should be ready for another review. Thanks!

**Comment by alexgibson** [comment]
The CircleCI failure above is unrelated to this change, and already fixed in master. This has been a long-running branch, so rather than delay further asking for another rebase, when we know things are ok, I'm going to squash & merge.

**Pull request merged**